src/schema/show.ts

```diff
@@ -495,6 +495,16 @@ export const ShowType = new GraphQLObjectType({
       args: pageable({
         sort: PartnerShowSorts,
         status: EventStatus,
+        displayable: {
+          type: GraphQLBoolean,
+          defaultValue: true,
```

Comment:
We are still defaulting to `displayable` when asking for a list of `nearbyShows` — but should we default to `discoverable` instead?

@orta would you know if there are any consumers besides the new LD screens that know or care about this field yet?

Reply:
Not to my knowledge

Reply:
Cool, I think I'll leave it as is. Seems more conservative, if the idea is not to leak out local discovery stub shows when they're not explicitly requested.

@mbilokonsky I agree about the weird subtleties among displayable/discoverable. I hoped to address that a bit by documenting [here](https://github.com/artsy/metaphysics/pull/1419/files#diff-7b1c236a12a473dffad8b241346f7f28R506) for consumers.